tests/distributions/conftest.py

```diff
@@ -183,6 +183,26 @@
             prec=0.08,
             is_discrete=True),
     Fixture(pyro_dist=(dist.categorical, Categorical),
+            scipy_dist=sp.multinomial,
+            examples=[
+                {'ps': [0.1, 0.6, 0.3],
+                 'test_data': [2]},
+                {'logits': list(map(math.log, [0.1, 0.6, 0.3])),
+                 'test_data': [2]},
+                {'logits': [list(map(math.log, [0.1, 0.6, 0.3])),
+                            list(map(math.log, [0.2, 0.4, 0.4]))],
+                 'test_data': [[2], [0]]},
+                {'ps': [[0.1, 0.6, 0.3],
+                        [0.2, 0.4, 0.4]],
+                 'test_data': [[2], [0]]}
+            ],
+            test_data_indices=[0, 1, 2],
+            batch_data_indices=[-1, -2],
+            scipy_arg_fn=lambda ps: ((1, np.array(ps)), {}),
+            prec=0.05,
+            min_samples=10000,
+            is_discrete=True),
+    Fixture(pyro_dist=(dist.categorical, OneHotCategorical),
```

Comment:
Change `dist.categorical` -> `dist.one_hot_categorical` so that the right functional form of the distribution is called.

Reply:
Done.